src/wireviz/DataClasses.py

```diff
@@ -43,6 +46,21 @@ def __post_init__(self, gv_dir):
                 if self.width:
                     self.height = self.width / aspect_ratio(gv_dir.joinpath(self.src))
 
+@dataclass
+class AdditionalComponent:
+    type: str
+    subtype: Optional[str] = None
+    manufacturer: Optional[str] = None
+    mpn: Optional[str] = None
+    pn: Optional[str] = None
+    qty: float = 1
+    unit: Optional[str] = None
+    qty_multiplier: Union[ConnectorMultiplier, CableMultiplier, None] = None
+
+    @property
+    def description(self) -> str:
+        return self.type.capitalize().strip() + (f', {self.subtype.strip()}' if self.subtype else '')
```

Comment:
This is needed to avoid some graphicial odities in the diagram. Without the strip if either string is a multiline string (defined with | as in the examples) a traling newline is added which causes issues when the fields are joined together in the cell. Most of the other cells ignore this problem as the trailing <br/> is ignored by graphvis.

with mini bom
![image](https://user-images.githubusercontent.com/3390465/96635111-5bc6e580-1313-11eb-8174-08f382c971bb.png)
without mini bom
![image](https://user-images.githubusercontent.com/3390465/96635252-85800c80-1313-11eb-9a89-469ba03ef24e.png)

The alternatives could be either getting user to use the |- symbol instead (should update the docs in this case). or moving the cleanup of the "<br />,", "<br />)" and "<br /><br />" into the render code for the additional components cells.

Reply:
In this case I'd argue having that little `.rstrip()` is nicer than requiring `|-` in the input YAML.

Reply:
My question is, where does the need for `.capitalize()` come from?
Perhaps I'm missing something, but I don't see a need to mess with the user-defined `type` attribute. This applies to all instances of this function call.

Reply:
The capitialise was originaly added to create more consistent bom formatting. We can remove this and pass that requirment onto the user if they care about it as this would be fairly intuative for a user to know what to change (unlike the newlines).

Reply:
I agree with your decision to remove the `.capitalize()`. Personally, I think I would have preferred recommending `|-` in the doc before removing the trailing newline with `.rstrip()`, but I understand that others might have a different opinion. The only motivation for a user to use `|` when specifying this attribute is to insert newlines, and then it shouldn't be too much of a surprise seeing the results above. If we in the doc write "recommend `|-` to avoid a trailing newline", that should be enough, IMHO, but I respect that you disagree. No hard feelings. 😄